### ISSUE TITLE
Watch some scripts (e.g. `build`) but not others (e.g. `serve`)

### ISSUE THREAD

**Issue by bennypowers** [google/wireit#265]
I've got a use case which I'll bet is quite common:
- a build process that outputs files, in my case `style-dictionary`
- a dev server with it's own file watcher, watching the files output in the previous step, in my case `@web/dev-server`

It wouldn't make sense to restart the dev server on changes, so using it's built-in filewatcher is smart.
We'd also like to be able to watch the build's input files for changes and restart the build whenever sources change
```json
"scripts": {
  "build": "wireit",
  "serve": "wireit",
  "start": "wireit"
},
"wireit": {
  "build": {
    "command": "style-dictionary build -c config.cjs",
    "files": [
      "tokens/**/*",
      "docs/**/*",
      "config.js"
    ],
    "output": [
      "build/**"
    ]
  },
  "serve": {
    "command": "wds --open --watch --root-dir build",
    "dependencies": [
      "build"
    ]
  },
  "start": {
    "???": "???"
  }
}
```

I want `npm run build` to run the build once, and I want `npm run start` to run the build command, then the dev server, then watch changes to the build command's inputs and rerun the build command when they change. I tried a few combinations of options but was unable to achieve that workflow with `wireit`.

I'd like to be able to define commands which are just combinations other commands, and I'd like to specify watchers in the config file, i.e. not with the `watch` command argument, but with some `"watch": ["build"]` in package.json

**Comment by rictic** [comment]
Related: #33

**Comment by aomarks** [comment]
Yeah this is https://github.com/google/wireit/issues/33 plus https://github.com/google/wireit/issues/237
